numpy/_core/src/multiarray/multiarraymodule.c

```diff
@@ -4327,6 +4327,101 @@ normalize_axis_index(PyObject *NPY_UNUSED(self),
 }
 
 
+static PyObject *
+_populate_finfo_constants(PyObject *NPY_UNUSED(self), PyObject *args)
+{
+    if (PyTuple_Size(args) != 2) {
+        PyErr_SetString(PyExc_TypeError, "Expected 2 arguments");
+        return NULL;
+    }
+    PyObject *finfo = PyTuple_GetItem(args, 0);
+    if (finfo == NULL || finfo == Py_None) {
```

Comment:
Can't be `NULL`, but doesn't matter.  I might not bother, it'll just say "can't set attributes of NoneType" without this, but happy either way.